scripts/apps/authoring/preview/field-types/urls.tsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+
+interface IProps {
+    urls: Array<{url: string; description: string}>;
+}
+
+export class UrlsPreview extends React.Component<IProps> {
+    render() {
+        return (
+            <div>
+                {
+                    this.props.urls.map(({url, description}) => (
+                        <div key={url}>{description}({url})</div>
```

Comment:
should urls be inside an `a` tag so the user can just click on it?

Reply:
It's meant to be printed so the link has to be inline. I have added a fix to make it clickable.